packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -210,8 +218,31 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
   async _verifySignature(
     commitData: CommitData,
     context: Context,
-    controller: string
+    controller: string,
+    streamId?: StreamID
   ): Promise<void> {
+    if (streamId) {
+      const protectedHeaders = commitData.envelope.signatures.map((s) => s.protected)
+      const decodedProtectedHeaders = protectedHeaders.map((pH) => base64urlToJSON(pH))
+      const capHeaderIndex = decodedProtectedHeaders.findIndex((dph) => !!dph.cap)
+      if (capHeaderIndex > -1) {
+        const capHeader = decodedProtectedHeaders[capHeaderIndex]
+        const capIPFSUri = capHeader.cap
+        const capCID = CID.parse(capIPFSUri.replace('ipfs://', ''))
+        const cacaoBlock = (await context.ipfs.dag.get(capCID)).value
```

Comment:
Does `(await context.ipfs.dag.get(capCID)).value` really return the block and not the value?

Reply:
My bad, this should be the `cacao`, not `cacaoBlock`